src/AppInstallerCLICore/Workflows/UninstallFlow.cpp

```diff
@@ -126,6 +127,18 @@ namespace AppInstaller::CLI::Workflow
             context.Add<Execution::Data::PackageFamilyNames>(packageFamilyNames);
             break;
         }
+        case InstallerTypeEnum::Portable:
+        {
+            auto productCodes = installedPackageVersion->GetMultiProperty(PackageVersionMultiProperty::ProductCode);
+            if (productCodes.empty())
+            {
+                context.Reporter.Error() << Resource::String::NoUninstallInfoFound << std::endl;
+                AICLI_TERMINATE_CONTEXT(APPINSTALLER_CLI_ERROR_NO_UNINSTALL_INFO_FOUND);
+            }
+            context.Add<Execution::Data::ProductCodes>(std::move(productCodes));
```

Comment:
There is no need to reuse this value.  And I would suggest that creating a context data item for the ARP entry object would be better anyway.

Reply:
Created `PortableARPEntry` context data item